backend/api/community_get.go

```diff
@@ -9,7 +9,7 @@ import (
 )
 
 type GetCommunityRequest struct {
-	ID int64 `uri:"id" binding:"min=0"`
+	ID int64 `uri:"id" binding:"required,min=0"`
```

Comment:
```suggestion
        ID int64 `uri:"id" binding:"required,min=1"`
```
small nit but technically id's cant be zero right?